app/src/main/res/values/strings.xml

```diff
@@ -21,6 +22,10 @@
     <string name="error_login_unknown">Unknown error logging in.</string>
     <string name="error_login_network">Network error logging in.</string>
 
+    <!-- logout -->
+    <string name="logout_dialog_title">Are you sure you want to log out?</string>
+    <string name="logout_dialog_message">You will need to log in again to enter or manage records after logging out. Logging in will require Internet access. Are you sure you want to log out now?</string>
```

Comment:
One of the source comments above implies that records which have not been uploaded to the server will be lost when logging out; if that's the case then I think this warning could be made a bit clearer.